client/test_helpers.go

```diff
@@ -34,3 +38,24 @@ func (t TestAccountRetriever) GetAccountNumberSequence(_ NodeQuerier, addr sdk.A
 	}
 	return acc.Num, acc.Seq, nil
 }
+
+// CallCliCmd calls theCmd cobra command and returns the output in bytes.
+func CallCliCmd(clientCtx Context, theCmd func() *cobra.Command, extraArgs []string) ([]byte, error) {
```

Comment:
Given that tests use and call `ApplyMockIO`, I personally don't see any merit to having an auxiliary function such as this. Too much abstraction IMHO.

Reply:
Yeah I don't disagree. Seems a bit superfluous to me too. @jgimeno are you in the position to provide a strong rationale to keep this function around?

Reply:
@jgimeno can you please update this? We can merge this after 👍

Reply:
Gonna do it right now

Reply:
My fault for introducing that. Seemed like a useful helper function. I guess it’s redundant